src/ape_test/providers.py

```diff
@@ -10,6 +10,15 @@
 
 
 class LocalProvider(TestProviderAPI, Web3Provider):
+
+    _tester: PyEVMBackend
+    _web3: Web3
+
+    def __init__(self, **data) -> None:
+
+        super().__init__(**data)
+        self.__post_init__()
```

Comment:
probably can just hoist everything in `__post_init__` to here, right?

Reply:
Does Pydantic have `__post_init__`?

Reply:
> Does Pydantic have `__post_init__`?

No, I don't believe so